### PR TITLE
VAGOV-1646: Event detail page query and template

### PR DESCRIPTION
**Pull request by acabouet**
## Description
Detail page for health care facility events. 

This requires some Drupal configurations to work: a new field type to generate .ics file download and then adding that field to events. Those configs are applied to va.gov-cms in this PR: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/187. 

## Screenshots
Screen Shot 2019-03-11 at 4 50 43 AM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/static-pages/sass/modules/_m-facility-events.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.va-c-event-info {
```
Comment (jbalboni) [inline src/applications/static-pages/sass/modules/_m-facility-events.scss]: Could we put a class on the `dl` elements instead, and then use `> dd` and `> dt` for those styles? In my experience writing styles like this puts you in place over time where we keep having to override styles with more and more selectors and it becomes hard to untangle or modify.

File: src/applications/static-pages/sass/modules/_m-facility-events.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    width: 33%;
+  }
+  dd {
+    margin-left: auto;
+    width: 66%;
+  }
+
+}
+
+.va-c-event-share {
```
Comment (jbalboni) [inline src/applications/static-pages/sass/modules/_m-facility-events.scss]: Same as above, could we create a class that goes on the `i` elements and style that?

File: src/site/stages/build/drupal/metalsmith-drupal.js [hunk 3]
```diff
@@ -305,6 +306,25 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
...
         );
         break;
+      case 'event': {
+        let addToCalendar;
+        for (const icsFile of icsFiles) {
+          if (
+            page.fieldAddToCalendar !== null &&
+            icsFile.fid === parseInt(page.fieldAddToCalendar.fileref, 10)
+          ) {
+            addToCalendar = icsFile.url;
```
Comment (jbalboni) [inline src/site/stages/build/drupal/metalsmith-drupal.js]: I'm not sure I understand this exactly, but are we only linking to one ics file here, while there could be multiple of them?

Comment (acabouet) [reply src/site/stages/build/drupal/metalsmith-drupal.js]: No there will only be one per event - it's generated with info from the event and then referenced via a file id on the event itself in fieldAddToCalendar. Here I'm using that id to find the actual file to be downloaded and then saving that file's url in a variable that i can print out on the template.

File: src/applications/static-pages/sass/modules/_m-facility-events.scss (hunk 1 above)
Comment (acabouet) [reply src/applications/static-pages/sass/modules/_m-facility-events.scss]: Moved this stuff to a class on the dl element.

File: src/applications/static-pages/sass/modules/_m-facility-events.scss (hunk 2 above)
Comment (acabouet) [reply src/applications/static-pages/sass/modules/_m-facility-events.scss]: Class is on the i element now.

**Review by jbalboni — APPROVED** [review]

**Review by aurora-a-k-a-lightning — APPROVED** [review]

**Pull request merged**